jspdf.plugin.cell.js

```diff
@@ -111,18 +112,21 @@
                         this.printHeaderRow(ln, true);
                     }
                 }
-                //We ignore the passed y: the lines may have diferent heights
-                y = (getLastCellPosition().y + getLastCellPosition().h);
-
+                
+                //We ignore the passed y: the lines may have different heights
+                // (Added to fix problem with a new page having an 'undefined' value for 'h'. This
+                // fix prevents the 'y' value below from being Nan and restarts table at 'y' on page.)
+                y = (getLastCellPosition().y + (getLastCellPosition().h || y));
```

Comment:
If the problem is with `getLastCellPosition().h` being undefined, why don't we change that to be `0` by default instead?

Also, what do you think of #387 fixing the same problem on a quite different way?